styles.css

```diff
@@ -95,3 +99,176 @@ img {
   left: 1.5rem;
   padding-top: 4rem;
 }
+
+/* work section */
+
+.work-container,
+.about-me {
+  display: grid;
+  grid-template-rows: repeat(1, 1fr);
+  grid-gap: 2rem;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+}
+
+.work-heading > h2 {
+  width: 327px;
+  height: 97px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  align-items: center;
+  text-align: center;
+}
+
+.line {
+  width: 48px;
+  height: 4px;
+  background: #36b37e;
+  border-radius: 24px;
+  margin: 0 auto;
+}
+
+.card-img {
+  background-color: #ebf0ee;
+  width: 100%;
+  height: 252px;
+}
+
+.grid-card {
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  margin-bottom: 1.5rem;
+}
+
+.box {
+  width: 327px;
+  height: 195px;
+  text-align: center;
+}
+
+.tags {
+  display: flex;
+}
+
+.tag-item {
+  margin-right: 0.5rem;
+  list-style: none;
+  text-decoration: none;
+  font-size: 14px;
+  padding: 2px 4px;
+  background: #ebf0ee;
+  border-radius: 4px;
+}
+
+.project-title {
+  color: #3a4a42;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+}
+
+button {
+  background: #36b37f;
+  border-radius: 4px;
+  color: #fff;
+  padding: 12px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border: none;
+}
+
+button:hover {
+  background-color: rgb(172, 165, 165);
+}
+
+button:enabled {
+  color: #172b4d;
+ }
```

Comment:
- [ ] Please implement the button hover and enabled states while following the Figma design guidelines shown below 👇
### Figma Design 
![image](https://user-images.githubusercontent.com/46839548/185723477-97767b35-9e69-4787-849e-65c469073642.png)
Kindly implement the `box-shadow` on the `button:hover`